man/memkind.3

```diff
@@ -313,14 +316,44 @@ is supported by TBB heap manager described in
 .B ENVIRONMENT
 section since Intel TBB 2019 Update 4.
 .PP
+.BR memkind_transfer_allocation ()
+implement allocation transfer inside specific
```

Comment:
You must describe this better.
... relocates the object if doing so could improve overall external fragmentation of the heap ...

Something to that effect.

Reply:
Done